TrafficLightsDemo/app_mingw.tcjs

```diff
@@ -10,10 +10,9 @@ tc.compilationMakeCommand = 'mingw32-make';
 
 tc.compilationMakeType = MakeType.GNU_make;
 tc.compileArguments = '$(DEBUG_TAG) -fexceptions  -DPOCO_STATIC';
-tc.compileCommand = '$(CC) ';
+tc.compileCommand = '$(CC)';
```

Comment:
But now this is the default value, so I think you can delete this whole line and it should still work. It's better to keep the TC file as short as possible without unnecessary lines.

Reply:
I did clean build by removing $(CC) I got compilation error.

Reply:
@rajneshkumarmalik That is very strange. In the other TC you don't have this line.
What compilation error did you get? What compile command was used when this line was removed?

Reply:
When I did clean build after removing $(CC) I got following error.
process_begin: CreateProcess failied
This issue is because 0_CC is initialized to empty in 0.mk file, which is used to generate the object files.

Reply:
@rajneshkumarmalik There is something strange going on here. I checked many other TCs and none of them has tc.compileCommand set to the default value '$(CC)'.
This line also I'm not sure about:
tc.compilationMakeCommand = 'mingw32-make';
Isn't the default 'make' in the path?

Reply:
When We install MinGW we get mingw32-make by default. We have to rename it to make. 
I think keeping mingw32-make is fine.